stage_descriptions/lists-04-sf6.md

```diff
@@ -38,13 +40,13 @@ The tester will execute your program like this:
 ./your_program.sh
 ```
 
-It will then create a new list with multiple elements.
+It will then create a new list with multiple elements:
 
 ```bash
 $ redis-cli RPUSH list_key "a" "b" "c" "d" "e"
 ```
 
-After that the tester will send your program a series of `LRANGE` commands. It will expect the response to be an RESP Array or empty array in each case, depending on the test case.
+After that, the tester will send your program a series of `LRANGE` commands. For each command, it will expect the response to be a [RESP Array](https://redis.io/docs/latest/develop/reference/protocol-spec/#arrays) or an empty array (`*0\r\n`), depending on the test case.
```

Comment:
```suggestion
After that, the tester will send your program a series of `LRANGE` commands. For each command, it will expect the response to be a [RESP array](https://redis.io/docs/latest/develop/reference/protocol-spec/#arrays) or an empty array (`*0\r\n`), depending on the test case.
```

(consistent with our usages in other places)